fluent-bit.yaml

```diff
@@ -0,0 +1,63 @@
+package:
+  name: fluent-bit
+  version: 2.0.8
+  epoch: 0
+  description: Fast and Lightweight Log processor and forwarder
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: Apache-2.0
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - bison
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - flex
+      - glibc
+      - libpq-15
+      - openssl-dev
+      - postgresql-15-dev
+      - yaml-dev
+      - zlib-dev
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://releases.fluentbit.io/2.0/source-${{package.version}}.tar.gz
```

Comment:
won't the 2.0 break when it moves to 2.1? Can we add a var here somehow?

Reply:
I mean I don't really know why it has to be in that sub-directory anyway - just because it was so we don't want to break existing builds. I can always update the Fluent Bit release process to make it top-level as the version is explicit in the package name anyway.

The other option is to use the Github Release/tag URL to get the source which was done originally.

Reply:
@amouat can we do some string manipulation here with melange? `2.0.8` => `2.0`?

Reply:
If you can that should be fine, I do something (disgustingly) similar for the release: https://github.com/fluent/fluent-bit/blob/7bcb502ebdcd647d6b83c972805e734ad522eb1a/.github/workflows/staging-release.yaml#L59-L61